fraqbot/prod-config.yaml

```diff
@@ -208,6 +208,26 @@ legos:
     path: Local.score_board.ScoreBoard
     kwargs:
       configs:
+        Worldle:
+          channels:
+            - wordle-variations
+          condition: '!contains([`0`, `1`], remainder(to_number(`${1}`), `7`))'
```

Comment:
I think this should be filtering out remainders 1 & 2.
Today (Sat, 99) % 7 = 1
Tomorrow (Sun, 100) % 7 = 2

Reply:
Addressed in https://github.com/ReformedDevs/fraqbot/pull/65/commits/0a96538a175e4e5388e752a41667bf4118690192